### DISCUSSION TITLE
using react-native-date-picker in expo

### DISCUSSION THREAD

**Discussion by irfo-cyper** [henninghall/react-native-date-picker#757] [Q&A]
will i be able to use this package in expo i tried and it throws 
Invariant Violation: requireNativeComponent: "RNDatePicker" was not found in the UIManager.
i am confused i am beginner

**Comment by Zajjaj-Khan** [comment]
Delete your Android and IOS folder and then 
`npx expo prebuild`
`npx expo run:android/ios`
